Userland/Services/SpiceAgent/Message.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2023, Caoimhe Byrne <caoimhebyrne06@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Message.h"
+#include <AK/MemoryStream.h>
+#include <AK/Stream.h>
+#include <AK/String.h>
+
+namespace SpiceAgent {
+
+ErrorOr<String> Message::debug_description()
+{
+    return TRY("Message {}"_string);
```

Comment:
What's going on here? that {} looks out of place 🤔

Reply:
> Should debug_description() be pure virtual?

Yeah, that would be better.